odoo/addons/base/models/ir_qweb_fields.py

```diff
@@ -394,6 +394,8 @@ def value_to_html(self, value, options):
             image = Image.open(BytesIO(base64.b64decode(value)))
             image.verify()
         except IOError:
+            if options.get('qweb_img_raw_data', False):
```

Comment:
why this ?

Reply:
I thought this was the only way to avoid a crash when trying to render the binary field if it contained a file and not an image (because I couldn't see how to differentiate a **file** binary field from an **image** binary field so my previous solution always added the `qweb_img_raw_data` attribute on the `t-field` regardless of the type of binary field)
But I found a way to know if a binary field is an image or not by checking if the model contains a field named like the field we're trying to add but with "_filename" at the end (e.g. if we are adding a field named "x_abc", the field is an image if the model doesn't contain a field named "x_abc_filename")